crowsnest/components/streamer/ustreamer.py

```
@@ -0,0 +1,165 @@
+            self._set_v4l2_ctrl(ctrl, postfix)
+        # Repulls the string to print current values
+        self.log_multiline(
+            self.cam.get_controls_string(),
+            logger.log_debug,
+            postfix=" v4l2ctl",
+        )
+
+    def _brokenfocus(self, focus_absolute_conf: str) -> None:
+        cur_val = self.cam.get_current_control_value("focus_absolute")
+        if cur_val and cur_val != focus_absolute_conf:
+            self.log_warning(f"Detected 'brokenfocus' device.")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)